apps/docs/src/app/core/component-docs/shellbar/shellbar-docs.component.html

```diff
@@ -45,20 +48,101 @@
 
 <separator></separator>
 
-<fd-docs-section-title [id]="'howItWorks'" [componentName]="'shellbar'">
-    How it works
+<fd-docs-section-title id="user-actions-menu" componentName="shellbar"> User actions menu </fd-docs-section-title>
+<description>
+    This example shows the user actions menu in shellbar. If no user thumbnail is available then display initials.
```

Comment:
rephrase to: 
Clicking on the profile picture opens the user actions menu. If no profile picture is set, a default icon is displayed with initials. Once clicked on the icon, options are vertically listed: [Recent Activities, Frequently Used, App Finder, Settings, Contact Support, Give Feedback, About, App Settings and Sign out]
<img width="279" alt="Screen Shot 2021-05-20 at 5 10 54 PM" src="https://user-images.githubusercontent.com/84039792/119049366-6e2efd80-b98e-11eb-9035-f6b2ca9223cd.png">
.